sockeye/lexicon.py

```diff
@@ -12,12 +12,13 @@
 # permissions and limitations under the License.
 
 import argparse
-import collections
 import operator
 import os
 import sys
+import time
 from typing import Dict, Generator, Tuple, Optional
 
+import collections
```

Comment:
Looks like an automatic tool accidentally moved `collections` out of the standard library import section.

Reply:
Actually thats the 'optimize import' function of PyCharm, which is supposed to standardize the order. I don't know exactly what the order is (maybe collections is not seens as a built-in module), but since we are using it heavily I'd like to rely on it. Are you ok with this?

Reply:
I checked the documentation for PyCharm and optimize imports is just supposed to follow the PEP8 style guide: https://www.python.org/dev/peps/pep-0008/#imports.  I haven't seen this issue before so it may be a one-off glitch.  What about manually editing it this time and revisiting if it happens again with other files?

Reply:
ok, will do